netjson-monitoring.lua

```diff
@@ -119,6 +119,29 @@ load_average = {tonumber(loadavg_output[1]),
                 tonumber(loadavg_output[2]),
                 tonumber(loadavg_output[3])}
 
+function parse_disk_usage()
+    file = io.popen('df')
+    disk_usage_info = {}
+    for line in file:lines() do
+        if line:sub(1, 10) ~= 'Filesystem' then
+            filesystem, size, used, available, percent, location = line:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
+            percent = percent:gsub('%W', '')
+            table.insert(disk_usage_info, {
+                filesystem = filesystem,
+                available_bytes = tonumber(available),
+                size_bytes = tonumber(size),
+                used_bytes = tonumber(used),
+                used_percent = tonumber(percent),
+                mount_point = location,
+        })
+        end
+    end
+    file:close()
+    return disk_usage_info
+end
+
+processors = io.popen('cat /proc/cpuinfo | grep -c processor')
```

Comment:
For coherency, please create a get_cpus function so that you can then add the data such as `cpus = get_cpus()`